src/Api/Controllers/TwoFactorController.cs

```diff
@@ -229,6 +229,7 @@ public async Task<TwoFactorWebAuthnResponseModel> GetWebAuthn([FromBody] SecretV
     }
 
     [HttpPost("get-webauthn-challenge")]
+    [ApiExplorerSettings(IgnoreApi = true)] // Disable Swagger due to CredentialCreateOptions not converting properly
```

Comment:
does this mean webauthn can't work for sdk without model revisions?

Reply:
There are a few options, but I think we would want to either hand-write the model in rust, or have our own request model.